readme.md

```diff
@@ -32,6 +32,13 @@ Default: `process.cwd()`
 
 Current working directory.
 
+#### options.args
+
+Type: `array`  
+Default: `arguments passed via cli`, e.g when running `xo --init` this defaults to `['--init']`
```

Comment:
I don't think the ```e.g when running `xo --init` this defaults to `['--init']``` is necessary and can cause confusion as `--init` is excluded internally.